.cmake-format.py

```diff
@@ -0,0 +1,21 @@
+# If a statement is wrapped to more than one line, than dangle the closing
+# parenthesis on it's own line.
+dangle_parens = True
+dangle_align = 'child'
+
+# If true, the parsers may infer whether or not an argument list is sortable
+# (without annotation).
+autosort = True
```

Comment:
can this be dangerous for the language lists?

Reply:
@3d-gussner probably knows.

I think the language list was always hand-sorted. The important part here is that we keep the two groups sorted separately, which is already the case.